tools/data-api/main.go

```diff
@@ -10,14 +10,18 @@ import (
 
 	"github.com/hashicorp/go-hclog"
 	"github.com/hashicorp/pandora/tools/data-api/internal/commands"
+	"github.com/hashicorp/pandora/tools/data-api/internal/logging"
 	"github.com/mitchellh/cli"
 )
 
 func main() {
-	logger := hclog.New(hclog.DefaultOptions)
-	logger.SetLevel(hclog.Trace)
+	loggingOpts := hclog.DefaultOptions
+	if v := os.Getenv("LOG_LEVEL"); v != "" {
+		loggingOpts.Level = hclog.LevelFromString(v)
+	}
+	logging.Log = hclog.New(loggingOpts)
 
-	logger.Info("Data API launched")
+	logging.Log.Info("Data API launched")
```

Comment:
_nit_ this one could become (so as to match everything else)

```suggestion
	logging.Infof("Data API launched")
```